ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.11.yaml

```diff
@@ -152,6 +152,13 @@ tests:
   steps:
     cluster_profile: openstack-hwoffload
     workflow: openshift-e2e-openstack-hwoffload
+- as: e2e-openstack-upi
```

Comment:
Looks like we're missing the 4.12 periodic jobs, we should generate them for shiftstack-ci (as a separate patch) and make the modification on the 4.12 file to match the commit message.

Reply:
good point. on it!